### PR TITLE
Fix wrong translation if using GridLayoutManager

### PR DESCRIPTION
**Pull request by paolovalerdi**
Hey! Thanks for this awesome library.

Currently if one is using GridLayoutManager [the x translation is always set to the last expanded item translation](https://imgur.com/a/VyNSfwx), this commit only adds that missing value in order to achieve a better animation.

Also, the initial alpha value was wrongly set to the expanded alpha so I also fixed it.

### CODE REVIEW

File: inboxrecyclerview/src/main/java/me/saket/inboxrecyclerview/page/ExpandablePageLayout.kt [hunk 1]
```diff
@@ -90,7 +90,7 @@ open class ExpandablePageLayout @JvmOverloads constructor(
 
   init {
     // Hidden on start.
-    alpha = expandedAlpha
+    alpha = collapsedAlpha
```
Comment (saket) [inline inboxrecyclerview/src/main/java/me/saket/inboxrecyclerview/page/ExpandablePageLayout.kt:93]: Whoops thanks for catching this!

File: inboxrecyclerview/src/main/java/me/saket/inboxrecyclerview/animation/ScaleExpandAnimator.kt [hunk 2]
```diff
@@ -0,0 +1,79 @@
+package me.saket.inboxrecyclerview.animation
+
+import android.view.View
+import me.saket.inboxrecyclerview.InboxRecyclerView
+import me.saket.inboxrecyclerview.page.ExpandablePageLayout
+
+/**
+ * Commented lines would allow to add scaling animation.
+ */
+class ScaleExpandAnimator : ItemExpandAnimator() {
```
Comment (saket) [inline inboxrecyclerview/src/main/java/me/saket/inboxrecyclerview/animation/ScaleExpandAnimator.kt]: Unrelated to this PR?

**Review by saket — CHANGES_REQUESTED** [review]

**Comment by paolovalerdi** [comment]
I just deleted that file by now, gonna work on it later :)

**Comment by saket** [comment]
Cool. `ScaleExpandAnimator` will be great to have!

**Pull request merged**